src-tauri/src/binaries/adapter_tor.rs

```diff
@@ -17,37 +17,37 @@ pub(crate) struct TorReleaseAdapter {}
 #[async_trait]
 impl LatestVersionApiAdapter for TorReleaseAdapter {
     async fn fetch_releases_list(&self) -> Result<Vec<VersionDownloadInfo>, Error> {
+        let dist_tor_bundle_url = "https://dist.torproject.org/torbrowser/13.5.6/tor-expert-bundle-windows-x86_64-13.5.6.tar.gz";
         let cdn_tor_bundle_url = "https://cdn-universe.tari.com/torbrowser/13.5.6/tor-expert-bundle-windows-x86_64-13.5.6.tar.gz";
-        let mut cdn_responded = false;
+        let mut dist_responded = false;
 
         for _ in 0..3 {
-            let response = reqwest::get(cdn_tor_bundle_url).await;
+            let response = reqwest::get(dist_tor_bundle_url).await;
```

Comment:
You are actually downloading the whole file here. Try a smaller file or use a method other than GET. Maybe HEAD or somethign?

Reply:
fixed